haystack/utils/preprocessing.py

```diff
@@ -154,14 +172,21 @@ def tika_convert_files_to_dicts(
                             last_para += " " + para
                         else:
                             if last_para:
-                                documents.append({"content": last_para, "meta": meta})
+                                documents.append(
+                                    Document(
+                                        content=last_para, content_type="text", meta=meta, id_hash_keys=id_hash_keys
+                                    )
+                                )
                             last_para = para
                     # don't forget the last one
                     if last_para:
-                        documents.append({"content": last_para, "meta": meta})
+                        documents.append(
+                            Document(content=last_para, content_type="text", meta=meta, id_hash_keys=id_hash_keys)
+                        )
+
         else:
             if clean_func:
                 text = clean_func(text)
-            documents.append({"content": text, "meta": meta})
+            documents.append(Document(content=text, content_type="text", meta=meta, id_hash_keys=id_hash_keys))
```

Comment:
`content_type="text",` is the default so there is no need to add it here. Only if you think we should make it explicit here.

Reply:
Same for all the other occurrences of `content_type="text"` in this file.